### PR TITLE
Do not bind mount /run/dbus to /run/dbus

### PR DESCRIPTION
**Pull request by pcarranzav**
connects to #208

---
This change is [Reviewable](https://reviewable.io/reviews/resin-io/resin-supervisor/210)

### CODE REVIEW

**Comment by pcarranzav** [comment]
@petrosagg

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+* Do not bind mount /run/dbus to /run/dbus [Pablo]
```
Comment (Page-) [inline CHANGELOG.md]: This should probably be marked as breaking/potentially breaking

**Comment by Page-** [comment]
SGTM

**Comment by petrosagg** [comment]
SGTM

**Pull request merged**